### PR TITLE
fix: Overlapping of name and url boxes in wizard 2.

### LINKED ISSUE #7011: Social Link Fields: Overlapping of name and url boxes in wizard 2.
**Describe the bug**

Overlapping of name and url boxes in wizard 2.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to create event wizard 2.
2. Click on add social media link.
3. See error 

**Expected behaviour**

 They should not overlap.

**Screenshots**

### PR DESCRIPTION
**Pull request by kapilchauhan8102**
Fixes #7011 

#### Short description of what this resolves:
This PR remove overlapping of name and url boxes in wizard 2.

#### Checklist

- [X] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [X] My branch is up-to-date with the Upstream `development` branch.
- [X] The acceptance, integration, unit tests and linter pass locally with my changes 
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by sachinchauhan2889 — DISMISSED** [review]
LGTM!

**Comment by iamareebjamal** [comment]
As this change is in core component, please add screenshots of other areas where it is used

**Comment by kapilchauhan8102** [comment]
@iamareebjamal  Sir, these are the screenshots of the other area where this component is used.

1. At exhibitor form.
   
exhibitor form
2. At session-speaker form.
   
speaker
session

3. At video room form.
 
video

I have checked all these working fine and all are exactly same as previously.

**Comment by iamareebjamal** [comment]
Those are link fields, not social link fields

**Comment by kapilchauhan8102** [comment]
> Those are link fields, not social link fields

@iamareebjamal  Sorry Sir for this.
 Finally these are the **screenshots of links field and social links field(before and after)** where this component is used.

There is **overlapping in exhibitor form also**, which is also solved.

### **Before Change**


**At wizard 2**

before change(wizard 2)

**At video room**

before change(video room)

**At speaker**

before change(speaker)

**At session**

before change(session)

**At exihibitor**

before change(exhibitor)2

before change(exhibitor)

### **After change**


**At wizard 2**

after change social link (wizard 2)

**At Speaker**

 after change (speaker)


 **At videoroom**

after change (video room)

**At session**

after change(session)

**At exhibitor** 

after change(exhibitor)

after change (exhibitor)

**Comment by iamareebjamal** [comment]
Now they have inconsistent widths.

**Comment by kapilchauhan8102** [comment]
@iamareebjamal  sir, sorry for disturbing you. Can you please ellaborate which widths?? 
i asked @sachinchauhan2889, even he is still confused.

File: app/components/widgets/forms/social-link-field.hbs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <div class="{{unless @fixed 'fields'}}">
   {{#unless @fixed}}
-    <div class="three wide field">
+    <div class="field pr-2">
```
Comment (maze-runnar) [inline app/components/widgets/forms/social-link-field.hbs:3]: don't remove three wide, just add pr-2.

**Comment by kapilchauhan8102** [comment]
@maze-runnar  Adding of three wide cause increase in gap between social link field and link field.
i.e., 


Screenshot from 2021-04-06 19-21-06

**Pull request closed without merging**